core/src/main/java/io/cloudevents/lang/Nullable.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.cloudevents.lang;
+
+import javax.annotation.Nonnull;
+import javax.annotation.meta.TypeQualifierNickname;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.*;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import static javax.annotation.meta.When.MAYBE;
+
+@Target(value = {METHOD, PARAMETER, FIELD})
+@Retention(value = RUNTIME)
+@Documented
+@Nonnull(when = MAYBE)
+@TypeQualifierNickname
+public @interface Nullable {
```

Comment:
This file belongs to `-api`

Reply:
yes but i still didn't introduced in this pr the dependency api -> core, so for now i need it there otherwise the code doesn't compile

Reply:
It really should be `core -> api`, not the other way around.

Reply:
Oh yes i'm sorry, i mistyped :smile: